### PR TITLE
Fixed the light color in bam view chart

### PR DESCRIPTION
**Pull request by YangQi007**
1, Fixed the light color in the global and sub-chromosome view of the bam chart 
2, Haven't solved the color changing when brushing a relatively big area.

### CODE REVIEW

File: coverage/src/BamViewChart.js [hunk 1]
```diff
@@ -240,25 +240,43 @@ function createBamView(bamHeader, data, element, bamViewControlsElement) {
                         .attr('transform', `translate(${margin.left}, ${mainHeight + margin.top + 20})`)
                         .attr('clip-path', 'url(#clip)');
 
+            const barWidth = 1; // Fixed bar width with 1 pixel
+            const numBins = innerWidth; // Number of bins based on inner width
+
+            // Aggregate data into bins
+            const aggregatedData = new Array(numBins).fill(0).map((_, i) => {
```
Comment (anderspitman) [inline coverage/src/BamViewChart.js:247]: This looks very similar to the `aggregateDataIntoBins` function. Are they the same? Can it be combined?

Comment (YangQi007) [reply coverage/src/BamViewChart.js]: Yes, After rethinking the code structure, I optimized the code. Thanks, Anders, for always pushing me to rethink the code quality.

Comment (anderspitman) [reply coverage/src/BamViewChart.js]: It's looking great

**Comment by anderspitman** [comment]
@YangQi007 do you still want to fix the brushing in a separate PR, or include it in this one?

**Comment by YangQi007** [comment]
Since they're related issues, I'll try to solve them in this PR.

**Comment by anderspitman** [comment]
Looks great @YangQi007! Excellent work.

**Pull request merged**